frontend/src/util/i18n.js

```diff
@@ -0,0 +1,121 @@
+const htmlDocPtTranslation = require('../../dist/app/html-document_pt_translation.json') || {}
+const threadEnTranslation = require('../../dist/app/thread_en_translation.json') || {}
+const threadFrTranslation = require('../../dist/app/thread_fr_translation.json') || {}
+const threadPtTranslation = require('../../dist/app/thread_pt_translation.json') || {}
+const fileEnTranslation = require('../../dist/app/file_en_translation.json') || {}
+const fileFrTranslation = require('../../dist/app/file_fr_translation.json') || {}
+const filePtTranslation = require('../../dist/app/file_pt_translation.json') || {}
+const folderEnTranslation = require('../../dist/app/folder_en_translation.json') || {}
+const folderFrTranslation = require('../../dist/app/folder_fr_translation.json') || {}
+const folderPtTranslation = require('../../dist/app/folder_pt_translation.json') || {}
+const adminWsUserEnTranslation = require('../../dist/app/admin_workspace_user_en_translation.json') || {}
+const adminWsUserFrTranslation = require('../../dist/app/admin_workspace_user_fr_translation.json') || {}
+const adminWsUserPtTranslation = require('../../dist/app/admin_workspace_user_pt_translation.json') || {}
+const wsAdvancedEnTranslation = require('../../dist/app/workspace_advanced_en_translation.json') || {}
+const wsAdvancedFrTranslation = require('../../dist/app/workspace_advanced_fr_translation.json') || {}
+const wsAdvancedPtTranslation = require('../../dist/app/workspace_advanced_pt_translation.json') || {}
+const wsEnTranslation = require('../../dist/app/workspace_en_translation.json')
+const wsFrTranslation = require('../../dist/app/workspace_fr_translation.json')
+const wsPtTranslation = require('../../dist/app/workspace_pt_translation.json')
+const agendaEnTranslation = require('../../dist/app/agenda_en_translation.json')
+const agendaFrTranslation = require('../../dist/app/agenda_fr_translation.json')
+const agendaPtTranslation = require('../../dist/app/agenda_pt_translation.json')
+const collaborativeDocumentEditionEnTranslation = require('../../dist/app/collaborative_document_edition_en_translation.json')
+const collaborativeDocumentEditionFrTranslation = require('../../dist/app/collaborative_document_edition_fr_translation.json')
+const collaborativeDocumentEditionPtTranslation = require('../../dist/app/collaborative_document_edition_pt_translation.json')
+const shareFolderEnTranslation = require('../../dist/app/share_folder_en_translation.json')
+const shareFolderFrTranslation = require('../../dist/app/share_folder_fr_translation.json')
+const shareFolderPtTranslation = require('../../dist/app/share_folder_pt_translation.json')
+const galleryEnTranslation = require('../../dist/app/gallery_en_translation.json')
+const galleryFrTranslation = require('../../dist/app/gallery_fr_translation.json')
+const galleryPtTranslation = require('../../dist/app/gallery_pt_translation.json')
```

Comment:
It seems this should be in a loop over the different languages, or even better, in two loops (over the languages, and over the apps). This code is going to be huge and difficult to extend as the number of apps and languages grow.

Reply:
indeed, it must be optimized

but I don't think this PR is the place for : (